### ISSUE TITLE
Build problem (test fails) in buildcomponents

### ISSUE THREAD

**Issue by espen42** [enonic/react4xp-npm#19]
Error message when running the `test` (or `testAll` since this task and `buildAll` are being renamed, but the issue existed before that) task in buildcomponents.

This also causes `npm run version` and the gradle task `npmLinks` to break, so this affects further development work, including dependency upgrades.

Error message:
```
Entrypoint react4xp-regions/Layout = templates.eb5f2adff.js react4xp-regions/Layout.js
Entrypoint react4xp-regions/Page = templates.eb5f2adff.js react4xp-regions/Page.js
[0] external "React" 42 bytes {1} {2} [built]
    + 2 hidden modules

ERROR in ./node_modules/react4xp-regions/entries/react4xp-regions/Layout.jsx
Module not found: Error: Can't resolve 'prop-types' in '/Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents/node_modules/react4xp-regions/entries/react4xp-regions'
 @ ./node_modules/react4xp-regions/entries/react4xp-regions/Layout.jsx 1:25-59 20:569-578 20:588-597 20:616-625 20:634-643 20:656-665 20:702-711 20:723-732 20:740-749 20:758-767 20:792-801 20:813-822 20:828-837 20:845-854 20:864-873 20:897-906 20:929-938 20:967-976

ERROR in ./node_modules/react4xp-regions/entries/react4xp-regions/Page.jsx
Module not found: Error: Can't resolve 'prop-types' in '/Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents/node_modules/react4xp-regions/entries/react4xp-regions'
 @ ./node_modules/react4xp-regions/entries/react4xp-regions/Page.jsx 1:25-59 20:563-572 20:592-601 20:611-620 20:639-648 20:657-666 20:679-688 20:725-734 20:746-755 20:763-772 20:781-790 20:815-824 20:836-845 20:851-860 20:868-877 20:887-896 20:928-937

ERROR in ./node_modules/react4xp-regions/entries/react4xp-regions/Layout.jsx
Module not found: Error: Can't resolve 'react4xp-regions/Regions' in '/Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents/node_modules/react4xp-regions/entries/react4xp-regions'
 @ ./node_modules/react4xp-regions/entries/react4xp-regions/Layout.jsx 1:59-105 20:421-428

ERROR in ./node_modules/react4xp-regions/entries/react4xp-regions/Page.jsx
Module not found: Error: Can't resolve 'react4xp-regions/Regions' in '/Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents/node_modules/react4xp-regions/entries/react4xp-regions'
 @ ./node_modules/react4xp-regions/entries/react4xp-regions/Page.jsx 1:59-105 20:422-429
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! react4xp-build-components@1.0.6 test:override: `webpack --config src/webpack.config.js --env.OVERRIDE_COMPONENT_WEBPACK=test/webpack.config.override.js --env.VERBOSE=true --env.REACT4XP_CONFIG_FILE=build/constants.json `
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the react4xp-build-components@1.0.6 test:override script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     (...) /2020-08-07T11_05_26_965Z-debug.log
```

(...) /2020-08-07T11_05_26_965Z-debug.log file dump:
```
0 info it worked if it ends with ok
1 verbose cli [ '/Users/espenjobb/.nvm/versions/node/v9.7.0/bin/node',
1 verbose cli   '/Users/espenjobb/.nvm/versions/node/v9.7.0/bin/npm',
1 verbose cli   'run',
1 verbose cli   'test:override' ]
2 info using npm@6.12.0
3 info using node@v9.7.0
4 verbose run-script [ 'pretest:override', 'test:override', 'posttest:override' ]
5 info lifecycle react4xp-build-components@1.0.6~pretest:override: react4xp-build-components@1.0.6
6 info lifecycle react4xp-build-components@1.0.6~test:override: react4xp-build-components@1.0.6
7 verbose lifecycle react4xp-build-components@1.0.6~test:override: unsafe-perm in lifecycle true
8 verbose lifecycle react4xp-build-components@1.0.6~test:override: PATH: /Users/espenjobb/.nvm/versions/node/v9.7.0/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents/node_modules/.bin:/Users/espenjobb/.sdkman/candidates/java/11.0.2-open/bin:/Users/espenjobb/.nvm/versions/node/v9.7.0/bin:/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/share/dotnet:~/.dotnet/tools:/Library/Apple/usr/bin:/Users/espenjobb/source/_npm_libs/react4xp-mono/node_modules/.bin:/Users/Shared/scripts:/Developer/python/current/bin
9 verbose lifecycle react4xp-build-components@1.0.6~test:override: CWD: /Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents
10 silly lifecycle react4xp-build-components@1.0.6~test:override: Args: [ '-c',
10 silly lifecycle   'webpack --config src/webpack.config.js --env.OVERRIDE_COMPONENT_WEBPACK=test/webpack.config.override.js --env.VERBOSE=true --env.REACT4XP_CONFIG_FILE=build/constants.json ' ]
11 silly lifecycle react4xp-build-components@1.0.6~test:override: Returned: code: 2  signal: null
12 info lifecycle react4xp-build-components@1.0.6~test:override: Failed to exec test:override script
13 verbose stack Error: react4xp-build-components@1.0.6 test:override: `webpack --config src/webpack.config.js --env.OVERRIDE_COMPONENT_WEBPACK=test/webpack.config.override.js --env.VERBOSE=true --env.REACT4XP_CONFIG_FILE=build/constants.json `
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (/Users/espenjobb/.nvm/versions/node/v9.7.0/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:127:13)
13 verbose stack     at ChildProcess.<anonymous> (/Users/espenjobb/.nvm/versions/node/v9.7.0/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:127:13)
13 verbose stack     at maybeClose (internal/child_process.js:936:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:220:5)
14 verbose pkgid react4xp-build-components@1.0.6
15 verbose cwd /Users/espenjobb/source/_npm_libs/react4xp-mono/packages/buildcomponents
16 verbose Darwin 19.6.0
17 verbose argv "/Users/espenjobb/.nvm/versions/node/v9.7.0/bin/node" "/Users/espenjobb/.nvm/versions/node/v9.7.0/bin/npm" "run" "test:override"
18 verbose node v9.7.0
19 verbose npm  v6.12.0
20 error code ELIFECYCLE
21 error errno 2
22 error react4xp-build-components@1.0.6 test:override: `webpack --config src/webpack.config.js --env.OVERRIDE_COMPONENT_WEBPACK=test/webpack.config.override.js --env.VERBOSE=true --env.REACT4XP_CONFIG_FILE=build/constants.json `
22 error Exit status 2
23 error Failed at the react4xp-build-components@1.0.6 test:override script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]
```

**Comment by espen42** [comment]
Reproduced both with locally linked node_modules (incl. webpack module.resolve.symlinks=true and false) and without. So that's probably not the issue.

**Comment by espen42** [comment]
Fixed in commit 1e90b500. Move on to versioning and QA.
